lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,22 @@
-
+require 'base64'
+require 'securerandom'
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
```

Comment:
Actually, it shouldn't - I have now cleaned up everything related to multi-use invite codes from this pull request because the rake task to generate invite codes didn't work in this branch. Now both PRs should be correct. Sorry about the confusion!